modules/Bio/EnsEMBL/Variation/DBSQL/VariationFeatureAdaptor.pm

```diff
@@ -1861,12 +1871,21 @@ sub fetch_by_hgvs_notation {
 
     my @transcripts;
 
+    # Fetch xref transcript
+    if(!defined($transcript) && $reference =~ /NP/){
+      @transcripts = @{$transcript_adaptor->fetch_all_by_external_name($reference_no_version)};
+    } 
+
     # support some malformed HGVS
     if(!defined($transcript)) {
       # Seeing transcripts erroneously submitted with p. changes
-      if($reference =~ /ENST|NM/){
+      if($reference =~ /ENST/){
```

Comment:
This needs to be backwards compatible as most species have not been reloaded in the new format - we need fetch_by_stable_id() for dog, for example, and fetch_all_by_external_name for human and maybe mouse.